lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -2547,12 +2547,28 @@ def _add_image_features(self, node_space: schema.NodeSpace) -> None:
         else:
             ...
 
-    def _load_image_features(self, node_space: schema.NodeSpace) -> None:
+    def _check_image_capability(self, node_space: schema.NodeSpace) -> None:
+        azure_runbook = node_space.get_extended_runbook(AzureNodeSchema, AZURE)
+
+        # If VHD doesn't enable AN, check if the requirement is supported
+        if azure_runbook.vhd and not azure_runbook.vhd.is_accelerated_network_supported:
+            synthetic_data_path = schema.NetworkDataPath.Synthetic
+            if node_space.network_interface:
+                matched_result = search_space.check_setspace(
```

Comment:
Just call `intersect`, and catch and rewrite the type of the Exception.

Reply:
Has called intersect_setspace_by_priority.

Reply:
About the comment "and catch and rewrite the type of the Exception", do you mean to write a new exception?

Reply:
It doesn't need a new exception, I just forget the behavior of intersect.